bin/subzero/runtime/src/lib.rs

```diff
@@ -158,7 +162,7 @@ impl_opaque_keys! {
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("subzero"),
-	impl_name: create_runtime_str!("dev"),
+	impl_name: create_runtime_str!("live"),
```

Comment:
subzero, not live

Reply:
Change `live` to `subzero`?

```rust
pub const VERSION: RuntimeVersion = RuntimeVersion {
	spec_name: create_runtime_str!("subzero"),
	impl_name: create_runtime_str!("subzero"),
	authoring_version: 75,
	spec_version: 63,
	impl_version: 0,
	apis: RUNTIME_API_VERSIONS,
	transaction_version: 1,
	state_version: 1,
};```

Something like this? So `spec_name` and `impl_name` refer to the same value?